### DISCUSSION TITLE
Force Sync with OneDrive server when in Monitor mode

### DISCUSSION THREAD

**Discussion by tyger** [abraunegg/onedrive#2268] [Q&A] [answered]
For the cases of app running as systemd service in monitor mode and very infrequent syncs (tens of minutes) sometimes it's valuable to qick off out of schedule sync with OneDrive server.
I'm curious if it is possible somehow? Maybe through sending singal to the process?

If not, then it would be very useful to have this option.

**Comment by abraunegg** [comment, marked as answer]
@tyger 
> I'm curious if it is possible somehow? Maybe through sending singal to the process?

> If not, then it would be very useful to have this option.

The only option is for you to manually stop the systemd service and perform the manual sync.

Sending a signal to an unknown process id, run by potentially a different user id, is not a viable option.
